src/group_theory/index.lean

```diff
@@ -105,37 +105,31 @@ begin
 end
 
 @[to_additive] lemma inf_relindex_right : (H ⊓ K).relindex K = H.relindex K :=
-begin
-  rw [←subgroup_of_map_subtype, relindex, relindex, subgroup_of, comap_map_eq_self_of_injective],
-  exact subtype.coe_injective,
-end
+by rw [relindex, relindex, subgroup_of_inf_left]
 
 @[to_additive] lemma inf_relindex_left : (H ⊓ K).relindex H = K.relindex H :=
 by rw [inf_comm, inf_relindex_right]
 
-@[to_additive relindex_inf_mul_relindex]
+@[to_additive]
```

Comment:
I thought that `to_additive` would default to `relindex_inf_add_relindex`, which is not what we want.

Reply:
Fixed, thanks!